pkg/ovs/openflow/ofctrl_group_test.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022 Antrea Authors
```

Comment:
```suggestion
// Copyright 2023 Antrea Authors
```
Please update the year in new files.